modin/_compat/core/latest/pandas_common.py

```diff
@@ -15,16 +15,49 @@
 
 from pandas.io.common import get_handle
 from pandas.core.apply import reconstruct_func
+from pandas import DataFrame as pandas_DataFrame
+from pandas.errors import DataError, SpecificationError
 
 
-def pd_pivot_table(df, **kwargs):  # noqa: PR01, RT01
+def pandas_pivot_table(df, **kwargs):  # noqa: PR01, RT01
     """Perform pandas pivot_table against a dataframe."""
     return df.pivot_table(**kwargs)
 
 
-def pd_convert_dtypes(df, **kwargs):  # noqa: PR01, RT01
+def pandas_convert_dtypes(df, **kwargs):  # noqa: PR01, RT01
     """Perform pandas convert_dtypes against a dataframe or series."""
     return df.convert_dtypes(**kwargs)
 
 
-__all__ = ["get_handle", "pd_pivot_table", "pd_convert_dtypes", "reconstruct_func"]
+def pd_compare(df, **kwargs):  # noqa: PR01, RT01
+    """Perform pandas compare against a dataframe or series."""
+    return df.compare(**kwargs)
+
+
+def pandas_dataframe_join(df, other, **kwargs):  # noqa: PR01, RT01
+    """Perform pandas DataFrame.join against a dataframe or series."""
+    return pandas_DataFrame.join(df, other, **kwargs)
+
+
+def pandas_reset_index(df, **kwargs):  # noqa: PR01, RT01
+    """Perform pandas reset_index against a dataframe or series."""
+    return pandas_DataFrame.reset_index(df, **kwargs)
+
+
+def pandas_to_csv(df, **kwargs):  # noqa: PR01, RT01
+    """Perform pandas to_csv against a dataframe or series."""
+    return df.to_csv(**kwargs)
+
+
+__all__ = [
+    "get_handle",
+    "pandas_pivot_table",
+    "pandas_convert_dtypes",
+    "pd_compare",
```

Comment:
```suggestion
    "pandas_compare",
```

Reply:
@vnlitvinov perhaps this message was lost among many others :)

Reply:
@vnlitvinov, ping

Reply:
fixed